include/rtc/peerconnection.hpp

```diff
@@ -139,8 +139,7 @@ class PeerConnection final : public std::enable_shared_from_this<PeerConnection>
 	void forwardMessage(message_ptr message);
 	void forwardMedia(message_ptr message);
 	void forwardBufferedAmount(uint16_t stream, size_t amount);
-	std::optional<std::string> getMidFromSSRC(SSRC ssrc);
-	std::optional<uint32_t> getMLineFromSSRC(SSRC ssrc);
```

Comment:
So I added this in as an optimization. Is it a good idea to be performing so many string compares in the real-time code?

Reply:
I think it's OK since the strings are guaranteed to be very small. The comparison is indeed a bit slower but it looks like an over-optimization since you will never have thousands of tracks.

Reply:
That's not necessarily true. I, or my peer, can name the mids whatever I wish. The last time I looked, `rtc::Description` defaults to setting the mid to `video` or `audio`.

Reply:
Yes, this is short, and the RFC actually recommends 4 characters or less. If there is a performance issue in some extreme use case, it would still be quick to add a SSRC to track cache, without re-implementing the mapping for line indexes.